src/main/java/org/apache/fineract/operations/TransactionRequest.java

```diff
@@ -75,6 +73,17 @@ public class TransactionRequest extends AbstractPersistableCustom<Long> {
     @Column(name = "SCENARIO")
     private String scenario;
 
+    @OneToMany(cascade = CascadeType.ALL, mappedBy = "transactionRequest")
```

Comment:
This should work with lazy fetch to make it performant.

Reply:
Done